src/samples/java/com/box/sdk/samples/appusers/LoginServlet.java

```diff
@@ -0,0 +1,61 @@
+package com.box.sdk.samples.appusers;
+
+import java.io.IOException;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.box.sdk.BoxAPIConnection;
+import com.box.sdk.samples.appusers.helpers.BoxHelper;
+
+/**
+ *
+ */
+public class LoginServlet extends HttpServlet {
+    private String connectMessage;
+    private Boolean loggedIn = false;
+
+
+    @Override
+    public void init() throws ServletException {
```

Comment:
Because global singleton initialization is only need once and login servlet is the first one called.